src/Components/common/validateFormTestimonials.js

```diff
@@ -0,0 +1,18 @@
+export const validateFormTestimonials = (values) => {
```

Comment:
this is a validations file, so the name of the file should be just validations. Then inside the validations files you can have a specific formTestimonialValidations